shigoto_q/tasks/enums.py

```diff
@@ -0,0 +1,6 @@
+import enum
+
+
+class TaskTypeEnum(enum.Enum):
+    REQUEST_ENDPOINT = "shigoto_q.tasks.tasks.request_endpoint"
+    KUBERNETES_JOB = "shigoto_q.tasks.tasks.k8s_job"
```

Comment:
added enum instead of hardcoding it everywhere, we should change this everywhere wheres its used
and perhaps use full name `...kubernetes_job` instead of `..k8s_job`